spec.html

```diff
@@ -6447,23 +6447,23 @@ <h1>
       <h1>
         CreateListFromArrayLike (
           _obj_: an ECMAScript language value,
-          optional _elementTypes_: a List of names of ECMAScript Language Types,
+          optional _elementTypes_: ~all~ or ~property-key~,
```

Comment:
This parameter actually gains nothing from being an enum. Though an enum is usually better, a boolean `validatePropertyKeys` or something would be just as clear here.